lua/telescope/_extensions/file_browser/make_entry.lua

```diff
@@ -185,6 +188,21 @@ local make_entry = function(opts)
     end
     if k == "stat" then
       local stat = vim.loop.fs_stat(t.value)
+      if not stat then
```

Comment:
I feel that's unnecessarily unwieldy.

What do you think about my original solution with simply bypassing file size and date for files w/o stat: https://github.com/nvim-telescope/telescope-file-browser.nvim/pull/139/files

I'd say we add logging to that and highlight files with no stat (for which something is wrong) with

```lua
table.insert(display_array, entry.stat and path_display or { path_display, "WarningMsg" })
```

as when there's not stat, there's not really anything meaningful (ie size, date, ...) to show anyways, or is there?

Reply:
I'm not super fond of it as it gives no real feedback to the user, but I completely agree that my solution is a bit big. 

I suppose though that this route doesn't really give the user much for discernable feedback either (they would have to know WHY the date is kicking back as NA which isn't terribly helpful), so maybe going with the simpler approach is the better option. I'm mildly annoyed that I can't figure out a better way to do it with metatables

Reply:
> I'm not super fond of it as it gives no real feedback to the user, but I completely agree that my solution is a bit big.

What kind of feedback would you like to give the user? Viable options I see -

1. Red highlighting as per my screenshot + clarifying in log not stat available - I personally like that one the most
2. `vim.notify` that no stat available -- off of the top of my head, I think is best implemented with a completion callback that clears itself after the first initial loop as in entry maker we don't have info about other entries 
3. Modify `entry.display` to e.g. show `f3 (no stat)` (possibly with 1.) - it's a bit unclear what "no stat" practically means (ie why is no stat available, which actually matters I'd say)

Conclusively, that's why I think the screenshot balances feedback with visual signal, as it's not perfectly clear why exactly we won't have an `stat` I think. 

> When I added this code snippet, I did not get any useful feedback on the UI. I toyed around a bit with it but couldn't get it to give me anything different than if I didn't include it. This is likely just my lack of knowledge of how Telescope works however

Did you set `t.stat = false` if `stat` is `nil` as per diff of my PR?

Reply:
> What kind of feedback would you like to give the user? Viable options I see -

**Red highlighting as per my screenshot + clarifying in log not stat available - I personally like that one the most**

I was attempting to get this but I could not until you dropped this little knowledge bomb on me
> Did you set t.stat = false if stat is nil as per diff of my PR? If it's still a table, they evaluate boolean to true, so it thinks there is a stat.

I was _not_ setting to false, I assumed an empty table failed a truthy check. Setting to `false` indeed makes the red highlighting work which is plenty of indication to the end user to me.

Also logs are good :)

I've updated the PR. Thanks for working with me on this!

Reply:
> I was attempting to get this but I could not until you dropped this little knowledge bomb on me

:laughing: 

> Thanks for working with me on this!

Sure thing :)

Final thing I think we have to edit. [telescope.log](https://github.com/nvim-telescope/telescope.nvim/blob/1a91238a6ad7a1f77fabdbee7225692a04b20d48/lua/telescope/log.lua#L3) only logs from `log.info(...)`. With `log.debug` as is, no message appears and no line is written to (for me) `~/.cache/nvim/telescope.log`. Once you set it to `info` or above, you'll get a message and the log entry. I guess `warn` would be the appropriate level?

Reply:
> Final thing I think we have to edit. [telescope.log](https://github.com/nvim-telescope/telescope.nvim/blob/1a91238a6ad7a1f77fabdbee7225692a04b20d48/lua/telescope/log.lua#L3) only logs from log.info(...). With log.debug as is, no message appears (while it should I think) and no line is written to (for me) ~/.cache/nvim/telescope.log. Once you set it to info or above, you'll get a message and the log entry. I guess warn would be the appropriate level?

Ahh ya I forgot to update that, I have all my various log levels set to debug currently, that is a good call. 

Updated the PR to reflect level warn (which I agree is probably the correct level)